samples/MiddlewareAnalysisSample/Startup.cs

```diff
@@ -85,6 +85,11 @@ public void Configure(IApplicationBuilder app, ILoggerFactory factory, Diagnosti
         public static void Main(string[] args)
         {
             var host = new WebHostBuilder()
+                .ConfigureLogging((_, factory) =>
```

Comment:
This is to configure `LoggerFactory` this overload will be called with the concrete `LoggerFactory`